### PR TITLE
#15 - Feature pdf destination

### PR DESCRIPTION
**Pull request by kpurrmann**
Set up TypoScript-Constant for using inline PDF

### CODE REVIEW

File: Classes/View/PdfView.php [hunk 1]
```diff
@@ -96,6 +99,9 @@ public function renderHtmlOutput($content, $pageTitle) {
             $pdf->SetHTMLFooter($this->getPartial('Footer', array('title' => $pageTitle)));
         }
 
+        $destination = 'attachment';
```
Comment (martin-helmich) [inline Classes/View/PdfView.php]: This first assignment is overwritten immediately.

File: Configuration/TypoScript/constants.txt [hunk 2]
```diff
@@ -24,6 +24,8 @@ plugin.tx_web2pdf {
 		pdfTopMargin = 15
 		# cat=plugin.tx_web2pdf/pdf/6; type=options[LLL:EXT:web2pdf/Resources/Private/Language/locallang.xlf:constants.pdf.styleSheet.print=print,LLL:EXT:web2pdf/Resources/Private/Language/locallang.xlf:constants.pdf.styleSheet.screen=screen]; label=LLL:EXT:web2pdf/Resources/Private/Language/locallang.xlf:constants.pdf.styleSheet
 		pdfStyleSheet = allAndPrint
+		# cat=plugin.tx_web2pdf/pdf/7; type=options[LLL:EXT:web2pdf/Resources/Private/Language/locallang.xlf:constants.pdf.destination.attachment=attachment,LLL:EXT:web2pdf/Resources/Private/Language/locallang.xlf:constants.pdf.destination.inline=inline]; label=LLL:EXT:web2pdf/Resources/Private/Language/locallang.xlf:constants.pdf.destination
+		pdfDestination = Download
```
Comment (martin-helmich) [inline Configuration/TypoScript/constants.txt]: Shouldn't this be `pdfDestination = attachment`?

File: Resources/Private/Language/locallang.xlf [hunk 3]
```diff
@@ -58,6 +58,15 @@
             <trans-unit id="constants.pdf.styleSheet.screen">
                 <source>screen</source>
             </trans-unit>
+            <trans-unit id="constants.pdf.destination">
+                <source>How should PDF be displaxed? Download or InBrowser?</source>
```
Comment (martin-helmich) [inline Resources/Private/Language/locallang.xlf]: Typo: `displa{x => y}ed`

**Comment by martin-helmich** [comment]
Sorry for the late review. Apart from some minor issues (see above), this look ok.

File: Configuration/TypoScript/constants.txt (hunk 2 above)
Comment (kpurrmann) [inline Configuration/TypoScript/constants.txt]: No, Download is the label of the option.

Comment (martin-helmich) [inline Configuration/TypoScript/constants.txt]: OK

**Comment by martin-helmich** [comment]
:+1:

**Pull request merged**